content/en/docs/pluggable-components/service-mesh.md

```diff
@@ -0,0 +1,150 @@
+---
+title: "Enable Service Mesh"
+keywords: "Kubernetes, istio, KubeSphere, service-mesh, microservices"
+description: "How to Enable KubeSphere Service Mesh"
+
+linkTitle: "Enable Service Mesh"
+weight: 3540
+---
+
+## What is KubeSphere Service Mesh
+
+On the basis of [Istio](https://istio.io/), KubeSphere Service Mesh visualizes microservices governance and traffic management. It features a powerful toolkit including **circuit breaking, blue-green deployment, canary release, traffic mirroring, distributed tracing, observability and traffic control**. Developers can easily get started with Service Mesh without any code hacking, with the learning curve of Istio greatly reduced. All features of KubeSphere Service Mesh are designed to meet users' demand for their business.
+
+For more information, see related sections in Project Administration and Usage.
+
+## Enable Service Mesh before Installation
+
+### Installing on Linux
+
```

Comment:
KubeSphere Service Mesh relies on Elasticsearch, so you are supposed to enable one of the three components (KubeSphere Logging System/KubeSphere Events System/KubeSphere Auditing System) before you enable the KubeSphere Service Mesh.